WalletWasabi.Fluent/WalletWasabi.Fluent.csproj

```diff
@@ -16,12 +16,12 @@
 		<AvaloniaResource Include="Assets\**" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="Avalonia" Version="0.10.5" />
-		<PackageReference Include="Avalonia.ReactiveUI" Version="0.10.5" />
-		<PackageReference Include="Avalonia.Diagnostics" Version="0.10.5" />
+		<PackageReference Include="Avalonia" Version="0.10.6" />
+		<PackageReference Include="Avalonia.ReactiveUI" Version="0.10.6" />
+		<PackageReference Include="Avalonia.Diagnostics" Version="0.10.6" />
 		<PackageReference Include="Avalonia.Xaml.Behaviors" Version="0.10.5" />
-		<PackageReference Include="Avalonia.Xaml.Interactions.Custom" Version="0.10.3" />
 		<PackageReference Include="Avalonia.Xaml.Interactivity" Version="0.10.5" />
```

Comment:
We can remove Avalonia.Xaml.Interactivity as it's redundant and is included already in Avalonia.Xaml.Behaviors